openedx/core/djangoapps/programs/tests/test_backpopulate_program_credentials.py

```diff
@@ -182,16 +178,15 @@ def test_handle_username_dedup(self, mock_task, mock_get_programs):
         mock_task.assert_called_once_with(self.alice.username)
 
     def test_handle_mode_slugs(self, mock_task, mock_get_programs):
-        """Verify that mode slugs are taken into account."""
+        """
+        Verify that course run types are taken into account when identifying
+        qualifying course run certificates.
+        """
         data = [
             factories.Program(
-                organizations=[factories.Organization()],
-                course_codes=[
-                    factories.CourseCode(run_modes=[
-                        factories.RunMode(
-                            course_key=self.course_id,
-                            mode_slug=MODES.honor
-                        ),
+                courses=[
+                    factories.Course(course_runs=[
+                        factories.CourseRun(key=self.course_id, type='honor'),
```

Comment:
i think I see your point, but at the same time if there are actually the same concepts would we ever want them to be different? maybe some day in the future we'll have an edx constants library or something